logfire/_internal/config.py

```diff
@@ -75,6 +74,7 @@
 from .exporters.quiet_metrics import QuietMetricExporter
 from .exporters.remove_pending import RemovePendingSpansExporter
 from .exporters.tail_sampling import TailSamplingOptions, TailSamplingProcessor
+from .exporters.test import TestExporter
```

Comment:
@Kludex it's not critical but I don't know why there isn't a stub for `exporters.test` when I run `rye run generate-stubs`. There's other stubs in the `exporters` folder.

Reply:
I think it's because testing is not in the `__init__.py`.

Reply:
I'm talking about `logfire._internal.exporters.test`, not `logfire.testing`

Reply:
Ah, because we don't include private objects.

Reply:
We'd need to add `--include-private` on the `stubgen` command for it to appear, I think...

Reply:
But all the other internal exporter files are there